Resources/config/services.xml

```diff
@@ -4,6 +4,8 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="Trikoder\Bundle\OAuth2Bundle\Service\ClientFinderInterface" alias="Trikoder\Bundle\OAuth2Bundle\Manager\ClientManagerInterface" />
```

Comment:
@toniperic This doesn't belong here IMO, I'd put it in `Resources/config/storage/doctrine.xml`.
@spideyfusion  WDYT?

Reply:
@HypeMC I think it belongs here due to the fact that if this was in `doctrine.xml` and someone configured the bundle to use the in-memory persistence method, then this alias wouldn't be registered at all.

Reply:
That a very good point, nice catch.